cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/datastore/Database200MigrationTest.java

```diff
@@ -100,7 +102,7 @@ public Void call(SQLDatabase db) throws Exception {
 
 
         int count = revisionCount();
-        Assert.assertThat(count, is(2));
+        Assert.assertThat("That a duplicate root has been removed.", count, is(2));
```

Comment:
"assert that that a duplicate root has been removed" ← code
"java.lang.AssertionError: That a duplicate root has been removed" ← message
neither of those look good to me